ccmlib/common.py

```diff
@@ -398,6 +400,8 @@ def scylla_extract_install_dir_and_mode(install_dir):
     elif install_dir.endswith('build/release') or install_dir.endswith('build/release/'):
         scylla_mode = 'release'
         install_dir = str(os.path.join(install_dir, os.pardir, os.pardir))
+    elif os.path.exists(os.path.join(install_dir, 'libreloc')):
+        scylla_mode = 'reloc'
```

Comment:
does reloc force to be release as well - e.g. is thre a reloc that is debug and a reloc that is release

Reply:
That's a good question, right now it's a mode of it's own, and they are all release, but in ccm this mode is basically used for some of the path finding. dtest has some code querying those mode, and selecting/deselecting tests based on it.

we might need to make the dtest a bit clear about this usage of the mode, or get it in some other mean.

as for a general question, do we want to publish multiple version of the relocatable ?, i.e. release/debug/dev ?

Reply:
On https://github.com/scylladb/scylla-pkg/pull/450 I support dev and debug.
Example build: https://jenkins.scylladb.com/view/master/job/scylla-master/job/manual-and-scheduled-tests/job/next_debug/426/ (failed on flaky tests, artefacts exist).

Reply:
I would like to have some information inside the tarball, to identify the build type, instead of different names or paths for those files. once we'll have that defined we'll update ccm with this ability

Reply:
@fruch please create an issue somewhere for this so we can merge this pull request and not assume you/hagit will remember this

Reply:
https://github.com/scylladb/scylla-ccm/issues/165